LiftLog.Ui/Models/SessionHistoryDao/SessionHistoryDaoV2.cs

```diff
@@ -111,14 +111,29 @@ public static PotentialSetDaoV2 FromModel(Lib.Models.PotentialSet model) =>
 
 internal partial class RecordedSetDaoV2
 {
-    public RecordedSetDaoV2(int repsCompleted, TimeOnlyDao completionTime)
+    public RecordedSetDaoV2(int repsCompleted, DateOnlyDao? completionDate, TimeOnlyDao? completionTime)
```

Comment:
```suggestion
    public RecordedSetDaoV2(int repsCompleted, DateOnlyDao completionDate, TimeOnlyDao completionTime)
```

We don't want to make this part nullable.  We should always have values by the time we get here.  It's purely in the `ToModel()` where it can be null